crates/bevy_pbr/src/render/pbr_prepass.wgsl

```diff
@@ -29,6 +29,11 @@ fn fragment(
 #endif
 
     var out: prepass_io::FragmentOutput;
+    
+#ifdef VERTEX_UVS
+    let uv_transform = material.uv_transform;
+    let uv = (uv_transform * vec3(in.uv, 1.0)).xy;
+#endif
```

Comment:
This is only used for the normal_prepass right? If so, it should probably be behind the ifdef of the normal prepass

Reply:
Correct. Good catch.